omop_file_validator.py

```diff
@@ -388,27 +390,22 @@ def run_checks(file_path, f):
 
                             for idx, value in df[submission_column].iteritems(
                             ):
+                                # convert pd nan values to empty string
+                                df = df.replace(np.nan, '')
```

Comment:
It looks like this is being done again. Maybe this can be done only once for the data frame in the beginning? Not sure how it would affect all usages.